README.md

```diff
@@ -6,69 +6,54 @@ Interested in our vision? Take a look at our [whitepaper](https://jolocom.io/wp-
 
 [![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/jolocom/SmartWallet)
 
-Setup for development
----------------------
+## Prerequisites
 
-Setup requires Node.JS to be installed on your computer. If you do not have it please see:
-https://nodejs.org/en/download/
+- Set-up requires [Node.js](https://nodejs.org/en/download/) to be installed on your computer.
+  - Starting with version `1.2.0` of the Jolocom SmartWallet, `Node.js v10+` is required to build the project. Versions prior to `1.2.0` require `Node.js v8`.
+- We use [Yarn](https://yarnpkg.com) as our package manager.
 
 ## Installation
 
-Prerequisites:
-- [yarn](https://yarnpkg.com)
+1. Clone the smartwallet-app repository to your computer.
+2. `cd` into the directory and run `yarn` or `yarn install` from your terminal to install the required packages.
 
-After cloning the smartwallet-app repository to a directory on your computer, enter the directory and run the following command:
-
-```bash
-yarn install
-```
-
-### Debug build on a device or simulator
-
-Starting with version `1.2.0`, the Jolocom SmartWallet requires `Node.js v10+` to build. Versions prior to `1.2.0` require `Node.js v8`.
+### Running a debug version for development
 
 #### Android
 
-Please set up an android development environment and install the required SDKs.
-Set the path to the SDK in ```android/local.properties``` and ```export``` it to the environment variable ```ANDROID_HOME```, if that was not already done.
+- Please set up an Android development environment and install the required SDKs.
+  - The [Getting Started](https://facebook.github.io/react-native/docs/getting-started) guide for React Native may come in handy.
+  - Look for the instructions under React Native CLI Quickstart.
+- Connect an Android device and enable USB debugging **OR** start an Android AVD emulator
+- Run `yarn run:android` to install the application and run it.
```

Comment:
Shouldn't there be a mention of running `yarn start` to run a bundler in a separate process intead of letting `yarn run:android` or `yarn run:ios` start one up automatically? The auto started ones have their stdout/stderr discarded, and that could be useful.